### DISCUSSION TITLE
Will NDR-240 fit in a 17x14x6?

### DISCUSSION THREAD

**Discussion by celestinjr** [tagyoureit/nodejs-poolController#937] [Q&A]
Trying to align on which enclosure size to go for, I'm looking to minimize the depth, given where my box will be located.  Can anyone confirm whether the Meanwell 240 series (i.e. SDR/NDR-240-24V) power supply will fit in the [17x14x6 enclosure referenced in the wiki](https://altelix.com/altelix-17x14x6-vented-din-rail-polycarbonate-abs-weatherproof-nema-enclosure/)?

I expected it would fit, since it's included in the wiki, but the [17x14x6 enclosure drawing](https://docs.altelix.com/drawings/DRW-NP171406VP.pdf) shows 105-107mm clearance from the face of the DIN rail, and the [NDR-240 drawing](https://www.meanwell-web.com/content/files/pdfs/productPdfs/MW/NDR-240/NDR-240-spec.pdf) says it's 113mm not including the DIN bracket.

Thanks!

**Comment by tagyoureit** [comment]
I believe @FlaMike went with the 17x14x6.  There have been quite a few threads about the correct size box.  https://github.com/tagyoureit/nodejs-poolController/discussions/871

**Comment by FlaMike** [comment]
Actually, I used the 14x12x8 box for my build.  In addition, I did not use
the transformer mentioned.  I used a rectifier bridge to get 24VDC from the
ET's 24VAC transformer.  The rectifier bridge is one of the differences in
my build vs. @CodeSlinger69's

Sorry I can't answer if the transformer will fit in the box in question.

**Reply by celestinjr** [reply]
@FlaMike, what rectifier bridge did you go with?

**Reply by FlaMike** [reply]
I have this rectifier bridge.  I tested both the input & output voltages several times with & without the Pi stack connected.  I had 27 VAC in & 24 VDC out.  So far it's steady as a rock & now it's less than I paid for it & has a 10% coupon, too.

Based on my experience date, I'd say go for it!

https://www.amazon.com/gp/product/B01JIKSHCA/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

Good luck & please post your experience.

**Reply by celestinjr** [reply]
Much appreciated. I was imagining you may have used something that also had some capacitor smoothing and regulated down to 24VDC.

I already have two rectifiers that the Aqualogic panel uses (same as the AquaRite), but I wasn’t sure if a rectified output would be clean enough to use directly.  I believe the rectified-only output would be a “pulsing” between 17-33VDC (average 24VDC).

**Comment by bluong77** [comment]
@celestinjr I purchase this enclosure from Amazon (Canada) "Joinfworld 15.7x11x5.9" Vented Outdoor Enclosure Box".
It fits an SDR-240-24.  I think they are similar size?

![IMG_2531 Large](https://github.com/tagyoureit/nodejs-poolController/assets/168031944/7f13cf41-99da-45bf-86f8-3eaddb3af3d8)


Link to enclosure here;
https://a.co/d/8PVdjTH

**Reply by celestinjr** [reply]
This helps, thanks!

**Reply by FlaMike** [reply]
I know you were expecting a capacitor to smooth the voltage swings.  Back when, like you, I inquired on these forums for bridge rectifier recommendations.  For whatever reason, no one responded.  Not being an electrical engineer, I did some reading on rectifiers but I could not locate a commercial (pre-built) one with "smoothing" of the DC current.  Eventually, I decided that a $10.00 bridge was hopefully superior to the chips on the ET motherboard & I'd give it a shot.  

I don't have an oscilloscope so I checked the output with a DMM.  It seemed happy enough with the output so I hooked up the Pi stack & everything worked on the bench, including the daughter boards.  I'm using the Sequent Computer hats recommended in the Wiki.  I believe the Building Automation hat has some voltage regulation and, as you probably know, can run on 24 VAC or 24 VDC.  I went the DC route per the Wiki to make the relay handing easier.  It's possible the Building Automation had cleans up the voltage from the bridge.

All I can say for certain is that 6 months in, everything seems happy.  We've had power outages, lightning storms, you name it, & the Nixie always comes up working.  Fingers crossed it continues to do so!

I hope this helps!!!
